### ISSUE TITLE
Local Auth issue: instructor list (in is_an_instructor?) contains "message"=>"Not Found".

### ISSUE THREAD

**Issue by mattscilipoti** [wdidc/assignments#9]
`is_an_instructor?` (in application_controller.rb), retrieves a list of instructors via:  
```
instructors = JSON.parse(HTTParty.get("https://api.github.com/teams/1511667/members?access_token=#{token}").body)
```
`token` is populated.

When performed locally, `instructors` contains:
```
{"message"=>"Not Found", "documentation_url"=>"https://developer.github.com/v3"}
```

**Comment by mattscilipoti** [comment]
I see that the readme says I "can generate an access token ... for testing purposes."  But, I don't know if I need to, nor what selections I should make.  I assumed I didn't, since `token` is populated.

**Comment by mattscilipoti** [comment]
I can reproduce this locally.  I have updated the code to raise the error - it was failing silently.

**Comment by jshawl** [comment]
`https://api.github.com/teams/1511667/members` will 404 if your current access token does not know that you are a member of the wdidc organization. visit github settings and make sure this application allows access to the wdidc organzation

**Comment by mattscilipoti** [comment]
After some spelunking, with Andy, we found that I had to look in the list of [Authorized Applications](https://github.com/settings/applications) and click on the Developer Application I setup for use by Assignments (you'll see that is is owned by you).  Mine was named "wdi-api-assignments".  This is the Developer Application where we got the Client ID and Client Secret for application.yml.  I had to authorize the wdidc organization for that (note: it was the only one that was not authorized).
